homeassistant/components/automation/websocket_api.py

```diff
@@ -69,6 +70,32 @@ def websocket_automation_trace_list(hass, connection, msg):
     connection.send_result(msg["id"], automation_traces)
 
 
+@callback
+@websocket_api.require_admin
+@websocket_api.websocket_command({vol.Required("type"): "automation/trace/subscribe"})
```

Comment:
This will listen to all trace updates. What's the use case for the UI ?

Reply:
That depends on the UI I guess.. Should we close this and open a new PR when the need is clearer?

Reply:
Yes. We should have our WS APIs be driven by the UI, not the other way around :)

Reply:
Generally we would have a draft PR open with a WS API while developing the UI. We can then keep changing it until we're happy.